### DISCUSSION TITLE
[@types/prismjs] Missing properties manual and disableWorkerMessageHandler

### DISCUSSION THREAD

**Discussion by srmagura** [DefinitelyTyped/DefinitelyTyped#58405] [Issues with a @types package]
On PR #58388, @RunDevelopment said:

> While the message from @DangerBotOSS can be ignored for the purpose of this PR, I still want to address its contents: Only the first 2 props are actually missing and are documented [here](https://prismjs.com/docs/Prism.html#.disableWorkerMessageHandler). `matchGrammar` is a purely internal function that is part of the Prism namespace for our (the Prism team) convenience.

I had actually already looked into adding the `manual` property, but it seems like it is not possible because variables exported by ES modules cannot be reassigned. So, even if the types had `export let manual: boolean | undefined`, it would not be possible to change `manual` unless `esModuleInterop=true`. And `@types` packages are not allowed to set `esModuleInterop=true` according to the README.

The situation is the same for `disableWorkerMessageHandler`.

Thoughts on fixing this, or just leave as is? Even if we leave as is, I can add a comment explaining why those 2 properties are not included.

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
@srmagura Please review #58417
